### PR TITLE
Update Iron Bank base image to 8.6

### PR DESCRIPTION
**Pull request by pugnascotia**
Iron Bank have advised us to upgrade the UBI base image.

### CODE REVIEW

**Review by mark-vieira — APPROVED** [review]
LGTM

**Review by jbudz — COMMENTED** [review]
https://github.com/elastic/elasticsearch/blob/master/distribution/docker/src/docker/iron_bank/hardening_manifest.yaml#L17 too?

Sorry for the random review - passing by while I update Kibana's too

File: docs/changelog/86796.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+pr: 86796
+summary: Update Iron Bank base image to 7.6
```
Comment (jbudz) [inline docs/changelog/86796.yaml:2]:
```suggestion
summary: Update Iron Bank base image to 8.6
```

**Comment by pugnascotia** [comment]
@jbudz good catch, thank you!

**Comment by lemongarbage** [comment]
Hello. I am re-opening this issue as the UBI base image is woefully outdated at this point as 8.6 is EoL. 

Can this please be updated to 8.8?

Thank you

**Comment by jbudz** [comment]
Hi, we merged https://github.com/elastic/kibana/pull/168219 for Kibana.  8.10.4 and 7.17.15 (not yet released) have been updated to UBI 9.2.

I'll leave the elasticsearch update to the team, I'm not familiar with the testing process in this repo.

**Comment by lemongarbage** [comment]
Hey there, thanks for replying.  Apologies for the mention, just trying to get some traction on this. 

OK 😄

**Comment by jbudz** [comment]
No problem! I forwarded the message along to see if we can find someone to take a look.

**Comment by mark-vieira** [comment]
@elastic/release-eng We're using the image at `docker.elastic.co/ubi8/ubi-minimal` is this no longer being update? @pugnascotia do you recall why we used this rather than the upstream image directly from registry1.dso.mil like Kibana is?

**Comment by alpar-t** [comment]
@mark-vieira it's updated daily, I'll send you the internal link over slack. But we're updating specific versions like 8.6 and 9.0 ...  AFAIK the reason we had this is that the images were not initially publicly available so I think the way to go is to use the images directly and remove the sync on our side.

**Comment by alsayasneh** [comment]
8.6 & 8.7 images are now EOL and 8.8 is the latest but will be EOL in the first quarter of 2024
So we should be moving to UBI 9.x, which is being updated more often by Red Hat with security fixes.

**Comment by lemongarbage** [comment]
Thank you all for your effort on this!

May I please ask that if possible, before moving to UBI 9, an intermittent release for UBI 8.8 is done?

**Comment by mark-vieira** [comment]
> AFAIK the reason we had this is that the images were not initially publicly available so I think the way to go is to use the images directly and remove the sync on our side.

Thanks, Alpar. I think we'll go with that and just use the upstream images like Kibana is doing.

**Comment by mark-vieira** [comment]
> May I please ask that if possible, before moving to UBI 9, an intermittent release for UBI 8.8 is done?

@lemongarbage could you explain the reason for this?

**Comment by lemongarbage** [comment]
@mark-vieira Purely my own use case. I have a need to specifically use UBI 8 and require the latest version of the base image. At this time I am not ready to take advantage of UBI 9.

**Pull request merged**